features/step_definitions/config_steps.rb

```diff
@@ -1,116 +1,108 @@
+# frozen_string_literal: true
 
-Given /^(provider "[^"]*") has config value "([^"]*)" set to (true|false)$/ do |provider, name, value|
+Given "{provider} has config value {string} set to true/false" do |provider, name|
```

Comment:
maybe this alternative text `option1/option2` can be used in steps above where `{word}` is presently used and the acceptable words are not apparent?